recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
There is no need for this when adding another item on the build matrix. All the build number `1` for Python `2.7` and `3.5` will be identical to build number `0`. (But it does not hurt to leave it there.)